build/yamls/windows/base/conf/Run-AntreaAgent.ps1

```diff
@@ -2,4 +2,4 @@ $ErrorActionPreference = "Stop"
 # wins will rename the binary when executing it. So we need to copy the binary everytime before running it.
 mkdir -force /host/k/antrea/bin
 cp /k/antrea/bin/* /host/k/antrea/bin/
-C:/k/antrea/utils/wins.exe cli process run --path /k/antrea/bin/antrea-agent.exe --args "--config=/k/antrea/etc/antrea-agent.conf --logtostderr=false --log_dir=/k/antrea/logs/ --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0" --envs "KUBERNETES_SERVICE_HOST=$env:KUBERNETES_SERVICE_HOST KUBERNETES_SERVICE_PORT=$env:KUBERNETES_SERVICE_PORT ANTREA_SERVICE_HOST=$env:ANTREA_SERVICE_HOST ANTREA_SERVICE_PORT=$env:ANTREA_SERVICE_PORT NODE_NAME=$env:NODE_NAME KUBE_DNS_SERVICE_HOST=$env:KUBE_DNS_SERVICE_HOST KUBE_DNS_SERVICE_PORT=$env:KUBE_DNS_SERVICE_PORT"
+C:/k/antrea/utils/wins.exe cli process run --path /k/antrea/bin/antrea-agent.exe --args "--config=/k/antrea/etc/antrea-agent.conf --logtostderr=false --log_dir=/var/log/antrea/ --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0" --envs "KUBERNETES_SERVICE_HOST=$env:KUBERNETES_SERVICE_HOST KUBERNETES_SERVICE_PORT=$env:KUBERNETES_SERVICE_PORT ANTREA_SERVICE_HOST=$env:ANTREA_SERVICE_HOST ANTREA_SERVICE_PORT=$env:ANTREA_SERVICE_PORT NODE_NAME=$env:NODE_NAME KUBE_DNS_SERVICE_HOST=$env:KUBE_DNS_SERVICE_HOST KUBE_DNS_SERVICE_PORT=$env:KUBE_DNS_SERVICE_PORT"
```

Comment:
If you didn't create the directory on the host, I think the antrea-agent process might fail to run because of the missing log directory

Reply:
Antrea on Windows supports two deploy solutions: 1) run in management Pod, 2) run as a host process (or Service). The logic in `Helper.psm1` is called when running as a process. But for management Pod, if you don't create the directory in `Install-WindowsCNI.ps1` or with the host volumn, I think the directory doesn't exist.

Reply:
Thanks for pointing this out. Pushed a new version that uses `type: DirectoryOrCreate` to resolve your concern.